doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/installing-ray.md

```diff
@@ -1,6 +1,112 @@
-:::{warning}
```

Comment:
this page is probably unnecessary,  we can link it to install ray main page

Reply:
Ok, that sounds good to me!